src/components/job/JobDashboard.jsx

```diff
@@ -0,0 +1,122 @@
+import { useRequest } from "ahooks";
+import { where } from "firebase/firestore";
+import { createContext, useContext, useEffect, useMemo } from "react";
+import { toast } from "react-hot-toast";
+import { NavLink, useParams } from "react-router-dom";
+
+import JobRoot from "@/components/job/JobRoot";
+import Spin from "@/components/Spin";
+import { JOB_STATUS } from "@/constants";
+import { useJob } from "@/models";
+
+const menu = [
+  { url: "pending", name: "All Pending Jobs" },
```

Comment:
use constants for url?

Reply:
no need. `constants` are for which are shared in multiple files